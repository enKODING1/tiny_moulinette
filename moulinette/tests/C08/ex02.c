#include <stdio.h>
#include <stdlib.h>
#include "ft_abs.h"

void test(int i)
{
	printf("MACRO ABS(%d): %d, abs(%d): %d\n", i, ABS(i), i, abs(i));
}

int main(void)
{
	test(-1);
	test(0);
	test(1);
	test(42);
	test(-2147483647);
	test(-2147483648);
	return 0;
}
