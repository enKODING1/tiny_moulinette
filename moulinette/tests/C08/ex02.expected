MACRO ABS(-1): 1, abs(-1): 1
MACRO ABS(0): 0, abs(0): 0
MACRO ABS(1): 1, abs(1): 1
MACRO ABS(42): 42, abs(42): 42
MACRO ABS(-2147483647): 2147483647, abs(-2147483647): 2147483647
MACRO ABS(-2147483648): -2147483648, abs(-2147483648): -2147483648
